google-cloud-bigquerystorage/clirr-ignored-differences.xml

```diff
@@ -212,5 +212,21 @@
     <className>com/google/cloud/bigquery/storage/v1/StreamWriter</className>
     <method>void setMissingValueInterpretationMap(java.util.Map)</method>
   </difference>
+  <difference>
+    <!--This class is marked as @InternalApi "public for technical reasons"-->
+    <differenceType>6004</differenceType>
+    <className>com/google/cloud/bigquery/storage/*/stub/readrows/ApiResultRetryAlgorithm</className>
+    <field>DEADLINE_SLEEP_DURATION</field>
+    <from>org.threeten.bp.Duration</from>
+    <to>java.time.Duration</to>
+  </difference>
+  <difference>
+    <!--The retryDelay field is used by ApiResultRetryAlgorithm, which is marked as @InternalApi-->
```

Comment:
Errors$IsRetryableStatusResult is still marked as public even if it's only used by `ApiResultRetryAlgorithm` in the BQStorage SDK. I think it's still accessible which is problematic.

Reply:
@PhongChuong what's the rationale of `Errors$IsRetryableStatusResult` being public? Is it similar to the case of `ApiResultRetryAlgorithm` "visible for technical reasons"?
Also, does it make sense to change their datatypes directly, without `threeten` alternatives as done currently?

Reply:
I am fine making it private. I don't think external user should use it.